web/modules/custom/bglobal_angular/components/info_sessions_list/component.ts

```diff
@@ -0,0 +1,53 @@
+import {Component} from "@angular/core";
+import {ElementRef} from '@angular/core';
+import {Inject} from '@angular/core';
+import {Http} from '@angular/http';
+import 'rxjs/add/operator/map';
+
+@Component({
+  moduleId: __moduleName,
+  selector: "info-sessions-list",
+  templateUrl: 'template.html'
+})
+
+export class InfoSessionsList {
+  public cert: string = 'all';
+  public baseUrl: string = 'https://voices.berkeley.edu/info-sessions/json/';
+  
+  // Constructor method sets our data from the JSON callback.
+  constructor(
+    @Inject(Http) http: Http,
+    @Inject(ElementRef) elementRef: ElementRef) {
+
+    this.setVariables(elementRef);
+    this.view = { count: 0 };
```

Comment:
`This.view` would be set outside the constructor unless there is a specific reason not to do so, in which case, that should be commented here.